### PR TITLE
Added load roster function and csv readers

### PR DESCRIPTION
**Pull request by shakibahmed35**
Im sorry Nick,
It can now read CSV files and send 2d arrays to the home page. 
it can also read attendance csv files and append them to the data 2d array

### CODE REVIEW

File: main/controller/ClickableOptions/LoadRosterOption.java [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
+import java.util.ArrayList;
 
 import javax.swing.AbstractAction;
 
 import main.controller.ClickableItem;
+import main.model.LoadRoster;
 import main.model.StateManager;
 
 public class LoadRosterOption extends ClickableItem{
+	LoadRoster rosterLoader;
```
Comment (nick-moran) [inline main/controller/ClickableOptions/LoadRosterOption.java]: delete >:(

File: main/model/StateManager.java [hunk 2]
```diff
@@ -11,6 +11,10 @@ public void updateState(String newState){
 		this.notifyObservers(newState);
 	}
 	
+	public void updateLoadRoster() {
```
Comment (nick-moran) [inline main/model/StateManager.java]: delete this >:(

File: main/view/MenuBar.java [hunk 3]
```diff
@@ -4,12 +4,14 @@
 import javax.swing.JMenuBar;
 
 import main.controller.ClickableOptions.*;
+import main.model.LoadRoster;
```
Comment (nick-moran) [inline main/view/MenuBar.java]: delete this >:(

File: main/view/MenuBar.java [hunk 4]
```diff
@@ -4,12 +4,14 @@
...
 
 import main.controller.ClickableOptions.*;
+import main.model.LoadRoster;
 import main.model.StateManager;
 
 public class MenuBar extends JMenuBar{
 	public MenuBar(StateManager state) {		
 		MenuOption fileMenu = new MenuOption("File");
 		MenuOption aboutMenu = new MenuOption("About");
+//		LoadRoster rosterLoader = new LoadRoster();
```
Comment (nick-moran) [inline main/view/MenuBar.java]: delete this >:(

**Review by nick-moran — CHANGES_REQUESTED** [review]
delete those :)

File: main/controller/ClickableOptions/LoadRosterOption.java (hunk 1 above)
Comment (shakibahmed35) [reply main/controller/ClickableOptions/LoadRosterOption.java]: Yes sIr 🖖

File: main/view/MenuBar.java (hunk 3 above)
Comment (shakibahmed35) [reply main/view/MenuBar.java]: yes boss man 😄

File: main/view/MenuBar.java (hunk 4 above)
Comment (shakibahmed35) [reply main/view/MenuBar.java]: I did 😃

File: main/model/StateManager.java (hunk 2 above)
Comment (shakibahmed35) [reply main/model/StateManager.java]: okie dokie my broski

**Review by nick-moran — APPROVED** [review]
👍

**Pull request merged**